packages/frontend/src/components/nft/NFTTransferModal.js

```diff
@@ -0,0 +1,397 @@
+import React, { useState } from 'react';
+import { Translate } from 'react-localize-redux';
+import { useDispatch, useSelector } from 'react-redux';
+import styled from 'styled-components';
+
+import { EXPLORER_URL } from '../../config';
+import { useFungibleTokensIncludingNEAR } from '../../hooks/fungibleTokensIncludingNEAR';
+import { checkAccountAvailable } from '../../redux/actions/account';
+import { clearLocalAlert, showCustomAlert } from '../../redux/actions/status';
+import { actions as nftActions } from '../../redux/slices/nft';
+import NonFungibleTokens, { NFT_TRANSFER_GAS } from '../../services/NonFungibleTokens';
+import isMobile from '../../utils/isMobile';
+import { formatNearAmount } from '../common/balance/helpers';
+import FormButton from '../common/FormButton';
+import Modal from '../common/modal/Modal';
+import ModalFooter from '../common/modal/ModalFooter';
+import SafeTranslate from '../SafeTranslate';
+import ReceiverInputWithLabel from '../send/components/ReceiverInputWithLabel';
+import AvatarSuccessIcon from '../svg/AvatarSuccessIcon';
+import EstimatedFees from '../transfer/EstimatedFees';
+
+
+const StyledContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0 0 0 0;
+
+    img {
+        width: 100% !important;
+        max-width: 472px;
+        border-radius: 8px;
+        margin-bottom: 16px;
+    }
+
+    .transfer-txt {
+        margin-top: 16px;
+    }
+
+    .confirm-txt {
+        margin-bottom: 4px !important; 
```

Comment:
Let's make sure we remove all usage of `!important` before we merge. You can add `&&& {...}` on `L24` to get the same specificity